src/app/builder/episode/episode-picker.component.ts

```diff
@@ -39,10 +39,22 @@ export class EpisodePickerComponent implements OnChanges, OnInit {
     this.select.emit(episode);
   }
 
+  private proxiedFeedUri() {
+    const customProxyUrl= window['ENV']['FEED_PROXY_URL'];
```

Comment:
I definitely like that as well. Hoisting it will also allow me to shorten this func - on it.